shorten_path_for_notational_fzf.py

```diff
@@ -65,20 +65,31 @@ def color(line, color):
 
 def process_line(line: str) -> str:
     # Expected format is colon separated `name:line number:contents`
+
+    # Windows paths may contain a colon, e.g. C:\Windows\ which messes up the split
+    # splitdrive(string) results in the following:
+    #   Windows drive letter, e.g. C:\Windows\Folder\Foo.txt -> ('C', '\Windows\Folder\Foo.txt')
+    #   Windows UNC path, e.g. \\Server\Share\Folder\Foo.txt -> ('\\Server\Share', '\Folder\Foo.txt')
+    #   *nix, e.g. /any/path/to/file.txt -> ('', '/any/path/to/file.txt')
+    #  Replace the : with a _ because the : will cause problems with FZF's delimiters, too.
+    drive, remainder = splitdrive(line)
+    line = drive.replace(':', '_') + remainder
     filename, linenum, contents = line.split(sep=":", maxsplit=2)
 
     # Drop trailing newline.
     contents = contents.rstrip()
 
     # Normalize path for further processing.
-    filename = abspath(filename)
+    if not platform.startswith('win32'):
```

Comment:
this should be floated into a global constant to avoid being re-calculated thousands of times since it runs on every line.

Reply:
Done!